pkg/rtc/mediatrack.go

```diff
@@ -427,7 +427,7 @@ func (t *MediaTrack) AddReceiver(receiver *webrtc.RTPReceiver, track sfu.TrackRe
 	}
 
 	var bitrates int
-	if len(ti.Layers) > int(layer) {
+	if layer >= 0 && len(ti.Layers) > int(layer) {
```

Comment:
panic in prod with layer `-1`

Reply:
ouch, will take a look at this.

This probably does not happen with pre-defined rid. Must be another case with dynamic rids.

Will bonk this PR, but also need to revert the pre-defined rids temporary change and update deps for next deploy. Will also look at what could be causing this edge case.